src/renderer.js

```diff
@@ -484,6 +484,16 @@ EPUBJS.Renderer.prototype.firstElementisTextNode = function(node) {
 	return false;
 };
 
+EPUBJS.Renderer.prototype.isGoodNode = function(node) {
+	"use strict";
```

Comment:
That's just a directive to make sure you don't, e.g., reference undeclared
global variables etc. in that function (
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode).
It's not necessary :)

2015-01-07 20:50 GMT+01:00 Fred Chasen <notifications@github.com>:

> In src/renderer.js
> <https://github.com/futurepress/epub.js/pull/183#discussion-diff-22610610>
> :
>
> > @@ -484,6 +484,16 @@ EPUBJS.Renderer.prototype.firstElementisTextNode = function(node) {
> >  	return false;
> >  };
> >
> > +EPUBJS.Renderer.prototype.isGoodNode = function(node) {
> > +	"use strict";
>
> What does "use strict" do here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/futurepress/epub.js/pull/183/files#r22610610>.
>